src/main/java/com/sem/btrouble/model/Wall.java

```diff
@@ -100,7 +100,7 @@ public void changeDirection() {
      */
     @Override
     public void draw(Graphics graphics) {
-        graphics.setColor(Color.black);
+        graphics.setColor(Color.blue);
```

Comment:
The wall drawing method is in the current state of the game just for debugging purposes (for checking where the walls are located), they don't necessarily have to be drawn, because the wall is already visible in the background. In order to make the wall more visible in front of the black background, i made them blue.